easy_rec/python/test/odps_run.py

```diff
@@ -231,6 +236,10 @@ def test_vector_retrieve(self):
     odps_oss_config.algo_res_project = args.algo_res_project
   if args.algo_version:
     odps_oss_config.algo_version = args.algo_version
+  algo_names = ['easy_rec_ext15', 'easy_rec_ext']
```

Comment:
should add usage in doc：https://easyrec.readthedocs.io/en/latest/quick_start/mc_tutorial.html